src/components/Histogram/HistogramComponent.tsx

```diff
@@ -109,7 +109,7 @@ export class HistogramComponent extends React.Component<WidgetProps> {
         return null;
     }
 
-    @computed get exportHeaders(): string[] {
+    private exportHeaders = (): string[] => {
```

Comment:
@TienHao could this be kept as a get function? I'm a bit confused by the change

Reply:
yes, I have reverted this change.